### ISSUE TITLE
power supply mod

### ISSUE THREAD

**Issue by wimpie007** [RubenKelevra/Tuya-thermostatic-valve#17]
Hi there,
nice work!
However, link to power supply mod is not available...

**Comment by RubenKelevra** [comment]
Yeah, I just need to find the images again. Super simple: Drill a hole to the battery compartment and solder a wire to the board from a 3V1A power supply. Works for month like a charm and is below 3 bucks :)

**Comment by DariBer** [comment]
Where on the board did you solder the plus and ground? Do you have a picture, photo?

**Comment by RubenKelevra** [comment]
Hey @DariBer,

on the back side of the PCB are the connections for the battery. They are marked with plus and minus and IIRC the wires are also red/black color coded. So I'm sure you'll find the right pads :)

**Comment by DariBer** [comment]
Found the JST connectors, and figured out which one is to the battery and what goes to the motor. 

I will crimp a cable with JST tomorrow when I get back my crimp tool
